ptcg_cockatrice.xml

```diff
@@ -990,7 +990,7 @@ Whenever you scry, you may draw a card. If you do, discard a card.</text>
 			<pt>1/3</pt>
 			<tablerow>2</tablerow>
 			<text>{1}{G}: Evolve
-Protection from Electric and Poison</text>
+Protection from Electric and instants</text>
```

Comment:
Why is it "protection from fighting and from normal" but then "protection from Electric and instants"? Just reads weird

Either way I vote we just give it protection from instants, could be cool. And make graveler 2/4

Reply:
Yeah it should be double from, that's my bad.